src/main/java/de/retest/recheck/review/ignore/PixelDiffFilter.java

```diff
@@ -76,7 +82,8 @@ private static String clean( final String str ) {
 
 	@Override
 	public String toString() {
-		return String.format( PixelDiffFilterLoader.FORMAT, pixelDiff );
+		final String value = specifiedAsFloat ? Double.toString( pixelDiff ) : Integer.toString( (int) pixelDiff );
+		return String.format( PixelDiffFilterLoader.FORMAT, value );
```

Comment:
I'm quite unsure, if I want specific formatting for serialization here or in the `Loader#save` (which per default delegates to `toString()`). 
Pro:
+ It separates the logic from the representation, since the filter actually does not care for the format.
+ The loader is responsible for the formatting and thus ensures the proper formatting upon saving.

Contra:
- It introduces somewhat duplicate code.
- We still need to have some kind of indicator inside/with the filter class (let it be a flag argument or instance-of check) for the loader to distinguish. See my comment above.

Ideas?

Reply:
Valid arguments, but I think this PR is the wrong place to discuss this. It fixes a bug, and as you already said, it currently delegates to `toString()` so it follows the given abstractions. Changing these abstractions should be done separately I would say?

Reply:
Yes, sure.